Docker.README.md

```diff
@@ -0,0 +1,41 @@
+# Supra DHT
+
+The Image exposes 3 ports `30333` `9933` `9944` which would need to be mapped to host ports appropriately.
+
+## Run node with _Alice_ as authority
+
+```bash
+docker run -it -p 30333:30333 -p 9944:9944 -p 9933:9933 supra-client:0.0.1
+```
+
+It executes the following command:-
+
+```bash
+supra-dht \
```

Comment:
Let’s just name it Supra this will ultimately be the foundation for the full client.

Reply:
Realized that after the checkin. Updating those now...